Tests/SentryTests/SentryCrash/SentryThreadInspectorTests.swift

```diff
@@ -80,6 +81,27 @@ class SentryThreadInspectorTests: XCTestCase {
         wait(for: [expect], timeout: 10)
     }
 
+    func testGetCurrentThreadWithStackTrack_TooManyThreads() {
+        let expect = expectation(description: "Wait all Threads")
+        expect.expectedFulfillmentCount = 70
+
+        let sut = self.fixture.getSut(testWithRealMachineContextWrapper: true)
+
+        for _ in 0..<expect.expectedFulfillmentCount {
+            Thread.detachNewThread {
+                expect.fulfill()
+                while self.fixture.keepThreadAlive {
+                    Thread.sleep(forTimeInterval: 0.001)
+                }
+            }
+        }
+
+        wait(for: [expect], timeout: 1)
```

Comment:
`m`: CI could be a bit slow sometimes to spawn 70 threads. I think we could be more generous with the timeout to avoid flakiness.
```suggestion
        wait(for: [expect], timeout: 5)
```